Cour-de-cassation/label
# Add zoning and occult zone

## Issue description :
- Get zoning and debatPublic metadata from sder database
- occult motivation if debat are not public

## Describe your changes :
- adding zoning and debatPublic to documentType
- creation of a new special categorie to occult motivation
- if decision debat are not public then create a special treatment to annotate motivations

## How to test :
- add zoning in storage or enable dbsder-api interraction to get TJ decisions with zoning directly from local sder database ( to enable dbsder-api : set `"db_api_enabled": true` in `localEnvironment.json` and use dbsder-api in the correct branch https://github.com/Cour-de-cassation/dbsder-api/tree/feat/call_zoning_on_create_decision)
- set debatPublic to false 
- check that motivation are well occulted after annotation
- try to find edge cases with overlapping annotations on motivations

## Checklist before requesting a review
> Occultation des motifs des décisions dont les débats ne sont pas publics. Cette occultation aura lieu dans une catégorie dédiée.
